packages/google-cloud-dataplex/google/cloud/dataplex_v1/services/data_scan_service/client.py

```diff
@@ -268,28 +268,6 @@ def parse_entity_path(path: str) -> Dict[str, str]:
         )
         return m.groupdict() if m else {}
 
-    @staticmethod
-    def table_path(
-        project: str,
-        dataset: str,
-        table: str,
-    ) -> str:
-        """Returns a fully-qualified table string."""
-        return "projects/{project}/datasets/{dataset}/tables/{table}".format(
-            project=project,
-            dataset=dataset,
-            table=table,
-        )
-
-    @staticmethod
-    def parse_table_path(path: str) -> Dict[str, str]:
-        """Parses a table path into its component segments."""
-        m = re.match(
-            r"^projects/(?P<project>.+?)/datasets/(?P<dataset>.+?)/tables/(?P<table>.+?)$",
-            path,
-        )
-        return m.groupdict() if m else {}
-
```

Comment:
This is a breaking change however the methods were only added [on July 25th](https://github.com/googleapis/google-cloud-python/pull/11512/files). In b/294420502, there is a note that this method would not have worked because the API only accepts
"//bigquery.googleapis.com/projects/{project}/datasets/{dataset}/tables/{table}" and not "projects/{project}/datasets/{dataset}/tables/{table}".  Approving based on the discussions in b/294420502